covalent/_results_manager/results_manager.py

```diff
@@ -155,13 +150,12 @@ def _get_result_from_dispatcher(
     adapter = HTTPAdapter(max_retries=5)
     http = requests.Session()
     http.mount("http://", adapter)
-    url = "http://" + dispatcher + "/api/result/" + dispatch_id
```

Comment:
any particular reason you removed the `f` string?

Reply:
It's [still there](https://github.com/AgnostiqHQ/covalent/pull/927/files#diff-5694ed1aa0e5cf6e57cf19bb998f92485940c7df6209fa3b0d719a33fbb38802R152).